taikun/resource_taikun_cloud_credential_openstack.go

```diff
@@ -26,6 +26,18 @@ func resourceTaikunCloudCredentialOpenStackSchema() map[string]*schema.Schema {
 			Type:        schema.TypeString,
 			Computed:    true,
 		},
+		"continent": {
```

Comment:
The following line should be added to [flattenTaikunCloudCredentialOpenStack (line 321)](https://github.com/nivraph/terraform-provider-taikun/blob/dev/taikun/resource_taikun_cloud_credential_openstack.go#L321) so that the Read operation can fetch the value of `continent`:
```go
"continent":                  rawOpenStackCredential.ContinentName,
```